### ISSUE TITLE
Clarification needed for kmer matrix columns 

### ISSUE THREAD

**Issue by Louis-MG** [tlemane/kmtricks#29]
Hello, I needed a matrix of the kmer counts for several samples. I followed the instructions given in the [example](https://github.com/tlemane/kmtricks/wiki/Count-matrix-example) of the documentation, but I don't see specified the correspondence between samples and columns, neither in the previous link nor in the documentation for the [aggregate](https://github.com/tlemane/kmtricks/wiki/Count-matrix-example) module. I would guess it follows the order given by the file of file ? I hope I didnt miss anything.
Thank you !

**Comment by nikostr** [comment]
I also looked for some info on this, and found the following at https://github.com/tlemane/kmtricks/wiki/IOs-API#1a-stream-it-1 

```
std::vector<count_type> counts(reader.infos().nb_counts); // count order follows the sample order in the input fof
```

When I've dumped the matrix previously it also behaves in a way that would be compatible with this.

**Comment by Louis-MG** [comment]
@nikostr Yeah I got the confirmation through a meeting with Teo that it is the case.
